src/state/apis/nft/parsers/alchemy.ts

```diff
@@ -84,29 +84,29 @@ export const parseAlchemyOwnedNftToNftItem = (
     id: collectionId,
     chainId,
     name:
-      alchemyOwnedNft.contract.name ||
-      alchemyOwnedNft.contract.openSea?.collectionName ||
+      alchemyOwnedNft.contract.name ??
+      alchemyOwnedNft.contract.openSea?.collectionName ??
       'Collection',
```

Comment:
haha, this one needs `||` since we want to treat empty strings as falsey values to use `Collection` as final fallback value

Reply:
faffa49220db426128d2e576518f6a9a29f20913